esx_service/vmdk_ops.py

```diff
@@ -402,8 +402,17 @@ def vol_info(vol_meta, vol_size_info, datastore):
 # Return error, or None for OK
 def removeVMDK(vmdk_path):
     logging.info("*** removeVMDK: %s", vmdk_path)
-    cmd = "{0} {1}".format(VMDK_DELETE_CMD, vmdk_path)
 
+    # Check the current volume status
+    attached, uuid, attach_as = getStatusAttached(vmdk_path)
+    if attached:
+        vm = findVmByUuid(uuid)
+        logging.info("*** removeVMDK: %s is in use by %s VM uuid = %s",
```

Comment:
On another host, the uuid will not translate to a VM object and can't be used. I was thinking more like the changes below,
  ---
    kv_status_attached, kv_uuid, attach_mode = getStatusAttached(vmdk_path)

    if kv_status_attached and kv_uuid != None:
        return err("Failed to remove %s - disk is in use by VM %s." % (vmdk_path, kv_uuid))
  ---